dd-bridge-android/src/main/kotlin/com/datadog/android/bridge/internal/BridgeSdk.kt

```diff
@@ -84,5 +90,21 @@ internal class BridgeSdk(
         )
     }
 
+    internal fun buildTrackingConsent(trackingConsent: String?): TrackingConsent {
+        return when (trackingConsent?.toLowerCase(Locale.US)) {
+            "pending" -> TrackingConsent.PENDING
+            "granted" -> TrackingConsent.GRANTED
+            "not_granted" -> TrackingConsent.NOT_GRANTED
+            else -> {
+                Log.w(
+                    BridgeSdk::class.java.canonicalName,
+                    "Unknown consent given: $trackingConsent, " +
+                        "using ${TrackingConsent.GRANTED} as default"
+                )
+                TrackingConsent.GRANTED
```

Comment:
not sure if default fallback should be `GRANTED`? Shouldn't we use `PENDING` instead? Anyway, DD SDK is using `GRANTED` if no explicit consent is given, but in my opinion it can be dangerous.

Reply:
Oh I didn't get that this was in case the value is unknown. In this case using PENDING might be a good option too since you add the warning in the logcat

Reply:
I would expect this to never happen, since I guess in all the language/technology-specific consumers of this bridge we can cover tracking consent with enum (like here https://github.com/DataDog/dd-sdk-reactnative/pull/40/files#diff-4058ddee4fced5d9a555912fe11b5564a89951f91bb9f4136982f60d013314d6R82), but so this guard is here mostly because of the incoming payload signature.

Reply:
implemented `PENDING` as fallback.